var/spack/repos/builtin/packages/openssl/package.py

```diff
@@ -186,6 +189,20 @@ def link_system_certs(self):
                     os.rmdir(pkg_certs)
                 os.symlink(sys_certs, pkg_certs)
 
+    @run_after('install')
+    def link_mozilla_certs(self):
+        if self.spec.variants['certs'].value != 'mozilla':
+            return
```

Comment:
I wish we could make the `@run_after` decorator work with the `@when` decorator, that would make this so much cleaner.